styleguide/source/_patterns/01-atoms/media/icons/svg/icon-calendar.twig

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Prob should run gulp minifyIcons to clean up the svg

Reply:
Running `gulp minifyIcons` doesn't change anything.